spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -199,6 +200,123 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
     }
   }
 
+  def windowExprToProto(
+      windowExpr: WindowExpression,
+      output: Seq[Attribute]): Option[OperatorOuterClass.WindowExpr] = {
+
+    val aggregateExpressions: Array[AggregateExpression] = windowExpr.flatMap { expr =>
+      expr match {
+        case agg: AggregateExpression =>
+          Some(agg)
```

Comment:
It would be better to fall back to Spark for aggregate expressions that we do not support, rather than fail during query execution.

```suggestion
        case agg: AggregateExpression => agg.aggregateFunction match {
          case _: Min | _: Max | _: Count => Some(agg)
          case _ => 
            withInfo(windowExpr, "Unsupported aggregate", expr)
            None
        }
```

Reply:
Fixed. Thanks!